contents/testing/security-testing.md

```diff
@@ -0,0 +1,93 @@
+<frontmatter>
+  title: Introduction to Security Testing
+  header: pagetop.md
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+<div class="website-content">
+
+{{ booktitle | safe }}
+
+# Introduction to Security Testing
+
+**Author(s): [Ahmed Bahajjaj](https://github.com/madanalogy)**<br>
+Reviewers: [James Pang](https://github.com/jamessspanggg)
+
+<box id="article-toc">
+
+* [Introduction to Security Testing](#introduction-to-security-testing)
+    * [What Is Security Testing?‎](#what-is-security-testing)
+    * [Why Have Security Testing?](#why-have-security-testing)
+    * [How to Get Started with Security Testing?](#how-to-get-started-with-security-testing)
+    * [Additional Resources‎](#additional-resources)
+</box>
+
+## What Is Security Testing?
+
+Security Testing is a type of Software Testing that uncovers vulnerabilities, threats, risks in a software application and prevents malicious attacks from intruders. 
+
+The goal of security testing is to:
+- Identify threats in the system.
+- To measure the potential vulnerabilities of the system.
+- To help in detecting every possible security risks in the system.
+- To help developers in fixing the security problems through coding.
```

Comment:
This list is a bit wobbly. Did you take it from some authoritative source? The four points use four terms: _threats_, _vulnerabilities_, _risks_, _problems_. Was that deliberate and with sound reasons behind the choices?